src/Util/Tokens.php

```diff
@@ -650,7 +650,7 @@ final class Tokens
      *
      * @var array <int|string> => <int|string>
      *
-     * @link https://www.php.net/language.constants.predefined PHP Manual on magic constants
+     * @see https://www.php.net/language.constants.predefined PHP Manual on magic constants
```

Comment:
I'd rather keep this at the correct tag (`@link`) than change it just too satisfy an outdated sniff. Could you please add an exception instead ?